recipe/meta.yaml

```
@@ -20,21 +20,20 @@ requirements:
     - wheel
   run:
     - python
-    - bokeh >=2.4.2,<3.0
-    - cytoolz >=0.8.2
+    - bokeh >=2.4.2
+    - click >=8.0
+    - cloudpickle >=1.5.1
     - dask-core {{ version }}
     - distributed {{ version }}
+    - fsspec >=2021.09.0
+    - importlib-metadata >=4.13.0
     - jinja2 >=2.10.3
-    - lz4
     - numpy >=1.21
-    - pandas >=1.3
-    - click >=7.0
-    - cloudpickle >=1.1.1
-    - fsspec >=0.6.0
     - packaging >=20.0
+    - pandas >=1.3
     - partd >=1.2.0
     - pyyaml >=5.3.1
-    - importlib-metadata >=4.13.0
+    - toolz >=0.10.0
```

```suggestion
    - toolz >=0.10.0
    - cytoolz >=0.10.0
```
cytoolz is a faster implementation of toolz